recipe/patch_yaml/README.md

```diff
@@ -0,0 +1,113 @@
+# repodata patch YAML specification
+
+The files in this directory are used to construct repodata patches for conda-forge.
+Typically, a single feedstock will have a single YAML file specifying the patches
+for the packages produced by that feedstock. Use comments liberally to describe
+why the patch exists.
+
+## YAML Format
+
+Patches are specified by two main blocks.
+
+- The `if` block specifies a set of conditions under which the changes in the `then` block are applied.
+- The different conditions in the `if` block are combined with a logical `AND`.
+- The `if` conditions can use shell glob syntax as implemented in the python `fnmatch` module in the
+  standard library.
+- Multiple patches can be in the same file using separate YAML documents (i.e., separate the data by `---`
+  on a new line).
+
+```yaml
```

Comment:
It would be nice to have this as `pydantic` model / JSON schema for easy validation before running the patch. I can contribute one if think it's a good idea!

Reply:
Yes please feel free to contribute directly to this PR!